test/E2ETests.cs

```diff
@@ -7,6 +7,8 @@
 namespace ReferenceTrimmer.Tests;
 
 [TestClass]
+[DeploymentItem("TestData/Directory.Build.props")]
```

Comment:
Isn't deployment item legacy / deprecated? You should instead just have them as content and read the files as if they were relative to the cwd

Reply:
There's no official info about it being legacy, and as you looks at the test code, it's just easier to read. Let's poke around for other ways to make it declarative.

Reply:
How about a data driven test where the data source is a property which provides the test name and the expectations? I forgot the name offhand but "data driven testing" in the docs is what to look for. "DataTestMethod" I think?

Reply:
Or just leave it as-is. I'm not convinced the existing pattern is even bad in any way.